src/components/TitleCard/index.tsx

```diff
@@ -130,14 +138,16 @@ const TitleCard = ({
           <div className="absolute left-0 right-0 flex items-center justify-between p-2">
             <div
               className={`pointer-events-none z-40 rounded-full border bg-opacity-80 shadow-md ${
-                mediaType === 'movie'
+                mediaType === 'movie' || mediaType === 'collection'
                   ? 'border-blue-500 bg-blue-600'
                   : 'border-purple-600 bg-purple-600'
               }`}
             >
               <div className="flex h-4 items-center px-2 py-2 text-center text-xs font-medium uppercase tracking-wider text-white sm:h-5">
                 {mediaType === 'movie'
                   ? intl.formatMessage(globalMessages.movie)
+                  : mediaType === 'collection'
+                  ? intl.formatMessage(globalMessages.movies)
```

Comment:
I think the badge should say "Collection" rather than "Movies" here -- there are languages where nouns don't have plural forms. Otherwise, this PR looks good to me!

Reply:
Also, to me, the difference is not really noticeable since the colour of the badge is the same.

Reply:
I think keeping the same color makes sense, since there are really just two types of requests in Overseerr, movies and series.